packages/types/src/users.ts

```diff
@@ -55,6 +55,7 @@ export type UserProfile = {
     terms_of_service_create_at: number;
     remote_id?: string;
     status?: string;
+    label?: string;
```

Comment:
I'd rather add an `AccessibleProfile` type alias to `components/widgets/inputs/users_emails_input.tsx` like
```tsx
type AccessibleProfile = UserProfile & {label: string}
```
than add to the type here. This property isn't applicable in other contexts, and doesn't represent what is fetched from the server. The packages types are for broadly applicable and reused definitions.